code/core/system/proxy-velodyne16/include/ProxyVelodyne16.h

```diff
@@ -23,69 +23,67 @@
+
+class ProxyVelodyne16 : public odcore::base::module::TimeTriggeredConferenceClientModule {
+   private:
+    /**
+     * "Forbidden" copy constructor. Goal: The compiler should warn
+     * already at compile time for unwanted bugs caused by any misuse
+     * of the copy constructor.
+     *
+     * @param obj Reference to an object of this class.
+     */
+    ProxyVelodyne16(const ProxyVelodyne16 & /*obj*/);
+
+    /**
+     * "Forbidden" assignment operator. Goal: The compiler should warn
+     * already at compile time for unwanted bugs caused by any misuse
+     * of the assignment operator.
+     *
+     * @param obj Reference to an object of this class.
+     * @return Reference to this instance.
+     */
+    ProxyVelodyne16 &operator=(const ProxyVelodyne16 & /*obj*/);
+
+   public:
+    /**
+     * Constructor.
+     *
+     * @param argc Number of command line arguments.
+     * @param argv Command line arguments.
+     */
+    ProxyVelodyne16(const int32_t &argc, char **argv);
 
-    class ProxyVelodyne16 : public odcore::base::module::TimeTriggeredConferenceClientModule {
-        private:
-            /**
-             * "Forbidden" copy constructor. Goal: The compiler should warn
-             * already at compile time for unwanted bugs caused by any misuse
-             * of the copy constructor.
-             *
-             * @param obj Reference to an object of this class.
-             */
-            ProxyVelodyne16(const ProxyVelodyne16 &/*obj*/);
+    virtual ~ProxyVelodyne16();
 
-            /**
-             * "Forbidden" assignment operator. Goal: The compiler should warn
-             * already at compile time for unwanted bugs caused by any misuse
-             * of the assignment operator.
-             *
-             * @param obj Reference to an object of this class.
-             * @return Reference to this instance.
-             */
-            ProxyVelodyne16& operator=(const ProxyVelodyne16 &/*obj*/);
+    odcore::data::dmcp::ModuleExitCodeMessage::ModuleExitCode body();
 
-        public:
-            /**
-             * Constructor.
-             *
-             * @param argc Number of command line arguments.
-             * @param argv Command line arguments.
-             */
-            ProxyVelodyne16(const int32_t &argc, char **argv);
+   private:
+    virtual void setUp();
+    virtual void tearDown();
 
-            virtual ~ProxyVelodyne16();
+    string memoryName;   //Name of the shared memory
+    uint32_t memorySize; //The total size of the shared memory: MAX_POINT_SIZE * NUMBER_OF_COMPONENTS_PER_POINT * sizeof(float), where MAX_POINT_SIZE is the maximum number of points per frame (This upper bound should be set as low as possible, as it affects the shared memory size and thus the frame updating speed), NUMBER_OF_COMPONENTS_PER_POIN=4 (x, y, z, intensity) Recommended values: MAX_POINT_SIZE=30000->ProxyVelodyne64.sharedMemory.size = 480000 in the configuration file.
```

Comment:
Same here (I don't comment on this further; please apply this to all class member variables in both subprojects).